sdk/src/realtime/composable.ts

```diff
@@ -76,7 +76,7 @@ export function realtime(config: WebSocketConfig = {}) {
 			}
 
 			// try filling in the defaults based on the main URL
-			const newUrl = new client.globals.URL(client.url.toString());
+			const newUrl = new client.globals.URL(client.url);
```

Comment:
`client.url` is still an `URL`.
```suggestion
			const newUrl = new client.globals.URL(client.url.toString());
```

Reply:
Yeah, but no need to transform it to a string here since `URL()` accepts `URL`

Reply:
True, It does accept any object with the `toString` function. I'd rather rely on explicitly passing a string than letting the URL constructor call the `toString` function (either way both situations are functionally equal one just a little more explicit imo)